certified-connectors/Oodrive Sign/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2891 @@
+          }
+        }
+      }
+    },
+    "/bundle/{id}/transaction/close": {
+      "get": {
+        "summary": "Validate contracts in a bundle",
+        "description": "Counter-sign (validate) all contract in a bundle",
+        "operationId": "ValidateContractsInBundle",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Bundle ID",
+            "description": "ID of the bundle to validate all contract",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "force",
+            "in": "query",
+            "required": false,
+            "type": "boolean",
+            "default": false,
+            "description": "Force the contracts validation if some signatories have not signed yet (or have refused to sign)",
+            "x-ms-summary": "Force validation"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "date": {
+                    "type": "string",
+                    "description": "Date of close state"
+                  },
+                  "id": {
+                    "type": "string",
+                    "description": "ID of the contract stopped",
+                    "x-ms-summary": "Contract ID"
+                  }
+                }
+              }
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/perimeters": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "List of perimeter",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Perimeter"
+              }
+            }
+          },
+          "404": {
+            "description": "Not found",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "List all perimeters",
+        "description": "Retrieve all perimeters associated to the connected environment",
+        "operationId": "ListAllPerimeters"
+      }
+    },
+    "/recipients": {
+      "get": {
+        "summary": "List all recipients",
+        "description": "Retrieve all the recipients associated to the current licence",
```

Comment:
@QwerioTEAM all files should be free of grammatical errors. please change "licence" to "license"